src/main/java/pl/trainingapp/repositories/ExerciseRepository.java

```diff
@@ -0,0 +1,38 @@
+package pl.trainingapp.repositories;
+
+import org.springframework.stereotype.Repository;
+import pl.trainingapp.entities.Exercise;
+import pl.trainingapp.exceptions.EntityNotFoundException;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public class ExerciseRepository {
+    
+    public void addExercise(String name) {
+        int maxId = ExerciseDatabase.exercises.stream().mapToInt(Exercise::getId).max().orElse(1);
```

Comment:
I guess its fine to do it this way until we add some kind of database, but you should add 1 to max, because otherwise you will have always the same ids

Reply:
yeah, I forgot about that